webapp/channels/src/sass/base/_structure.scss

```diff
@@ -234,8 +241,13 @@ body.app__body #root {
             padding-right: 0;
             z-index: 17;
             grid-template:
-                'announcement'min-content 'header'min-content 'main'auto 'footer'min-content / auto;
-
+                'announcement' min-content
+                'admin-announcement' min-content
```

Comment:
This is the main change. Other changes are such that its easier to see the columns and rows in grid layout.